docs/variants/novacustom_nv4x_adl/heads.md

```diff
@@ -0,0 +1,164 @@
+* coreboot revision [3a9aa3a4](https://github.com/Dasharo/coreboot/commits/3a9aa3a4692f3dd49732f5b4e3ec54be385f0969/)
+  based on upstream version [v4.21](https://doc.coreboot.org/releases/coreboot-4.21-relnotes.html)
+
+### Known issues
+
+* [Power button does not work in Qubes](https://github.com/Dasharo/dasharo-issues/issues/710)
+* [Heads shuts down instead of rebooting](https://github.com/Dasharo/dasharo-issues/issues/711)
+* [Existing Qubes installation is not bootable after transition back to EDK2](https://github.com/Dasharo/dasharo-issues/issues/713)
+
+### SBOM (Software Bill of Materials)
+
+* [Dasharo coreboot fork rev 3a9aa3a4 based on upstream version 4.21](https://github.com/Dasharo/coreboot/tree/3a9aa3a4)
+* [Dasharo Heads fork rev ccf49703 based on upstream rev 25066e5f](https://github.com/Dasharo/heads/tree/ccf49703)
+
+## Hardware Configuration Matrix
+
+Generally, the same [hardware configuration](hardware-matrix.md) as for the
+UEFI variant applies.
+
+A notable addition is usage of the
+[Nitrokey 3A Mini](https://novacustom.com/product/nitrokey-3a-mini/)
+USB device, which is required for Heads installation and usage.
+
+## Test Matrix
+
+Please refer to the [tests results spreadsheet](https://docs.google.com/spreadsheets/d/1yWZ--zFPIsQhXZByf7nJIrasQYuRSf1yCi60lY_RGsQ).
+
+## Building manual
+
+This section presents the crucial steps required to build the Dasharo Heads
+firmware. For more information, you may also refer to the official
+[Heads building documentation](https://osresearch.net/general-building/).
+
+### Requirements
+
+This guide was verified on Ubuntu 22.04. In practice, any Linux distribution
+with [Docker](https://www.docker.com/) support should be enough to complete it.
+
+Make sure that you have following packages installed:
+
+* Docker
+    - follow [Install Docker Engine on Ubuntu](https://docs.docker.com/engine/install/ubuntu/)
+    - follow [Post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/)
+* Git
+
+    ```bash
+    sudo apt -y install git
+    ```
+
+### Building
+
+1. Clone Dasharo Heads repository:
+
+    ```bash
+    git clone https://github.com/Dasharo/heads.git
+    ```
+
+1. Navigate to the source code directory and checkout to the desired revision:
+
+    ```bash
+    cd heads
+    git checkout novacustom_nv4x_adl_v0.9.0
+    ```
+
+1. Start the build inside the docker container:
+
+    ```bash
+    docker run --rm -it -v $PWD:$PWD -w $PWD \
+      3mdeb/heads-docker:3.0.1 make BOARD=nitropad-nv41
+    ```
+
+This will produce a Dasharo binary placed in
+`build/x86/nitropad-nv41/dasharo-nitropad-nv41-*.rom`.
+
+## Switching from Dasharo UEFI to Heads
+
+To change firmware branches from UEFI to Heads, because of how different the two
+firmware types are, it's required to disable some security measures before
+flashing. Follow the steps below to install Heads from an existing Dasharo UEFI
+firmware installation:
+
+* Enter UEFI Setup Menu and disable:
```

Comment:
I understand that Heads firmware  is designed for advanced users, but could you please share a quick guide on accessing the menu? I believe it's `F2` for NV4XPZ.

For naming consistency, users should head to `Dasharo System Features`  menu and then `Dasharo Security Options`  menu. Please verify, and use the exact names we have in our firmware.

Reply:
Added more explicit steps: https://github.com/Dasharo/docs/pull/754/commits/40abb8386a217b149422e72be135d21b82cdeed5